mutator/branch/mutateif.go

```diff
@@ -20,7 +22,13 @@ func MutatorIf(pkg *types.Package, info *types.Info, node ast.Node) []mutator.Mu
 	}
 
 	old := n.Body.List
-
+	
+	containsErr := strings.Contains(fmt.Sprintf("%v", n.Cond), "err")
+	containsNilCheck := strings.Contains(fmt.Sprintf("%v", n.Cond), "!= nil")
+	
+	if containsErr && containsNilCheck && len(n.Body.List) == 1 {
```

Comment:
what if the body has something like the following:

```go
if err != nil {
  err = fmt.Errorf("foo bar: %v", err)
  return err
}
```

Reply:
The real question is are there any cases where an error block like this is _not_ a trivial mutation? If not, why don't we just change the length condition to len = 1 or 2?

Reply:
imo lets run into this some more and see what happens. I'd rather have false positives flagged once or twice and we omit them, then remove actually helpful mutations in places

Reply:
agreed!